src/conditional-flow/numeric-conditions.js

```diff
@@ -1,28 +1,36 @@
 // TODO: Implement the functions below to make the tests pass
 
 // This function should return true if there are no elements in the array, false otherwise
-function isArrayEmpty (array) {
-
+function isArrayEmpty(array) {
+  return array.length == 0 ? true : false;
   // TODO: write code in this function body to pass the tests
-
 }
 
 // This function should return true if num1 is greater than num2, false otherwise
-function isGreaterThan (num1, num2) {
-
+function isGreaterThan(num1, num2) {
   // TODO: write code in this function body to pass the tests
-
+  if (num1 > num2) {
```

Comment:
The if statement is not needed here as `num1 > num2` is already a boolean condition, so you can just return that value such as: 
```js
return num1 > num2
```